Source/WebCore/platform/graphics/skia/PlatformDisplaySkia.cpp

```diff
@@ -66,6 +66,33 @@ static sk_sp<const GrGLInterface> skiaGLInterface()
 
 static thread_local RefPtr<SkiaGLContext> s_skiaGLContext;
 
+int initializeMSAASampleCount(GrDirectContext* grContext)
+{
+#if PLATFORM(GTK)
+    // GTK is desktop oriented, where using the GPU for antialiasing is not that necessary.
+    return 0;
+#endif
+
+    static std::once_flag onceFlag;
+    static int sampleCount = 0;
+
+    std::call_once(onceFlag, [grContext] {
+        const char* disableMSAA = getenv("WPE_DISABLE_MSAA");
```

Comment:
I would rather make the sample count configurable, aka WEBKIT_MSAA_SAMPLE_COUNT. If unset, on WPE, assume 4. On Gtk, if unset assume 0.